content/docs/1.4.0/advanced-resources/rwx-workloads.md

```diff
@@ -31,6 +31,41 @@ It is necessary to meet the following requirements in order to use RWX volumes.
     
     > **Tip:** The [environment check script](https://raw.githubusercontent.com/longhorn/longhorn/v{{< current-version >}}/scripts/environment_check.sh) helps users to check all nodes have unique hostnames.
 
+# Notice
```

Comment:
Here is the suggested wording:

# Known Issue

In versions 1.4.0 to 1.4.3 and 1.5.0 to 1.5.1, Longhorn volumes that are exported by an NFS server hosted within a share manager pod (specifically, in `NodeStageVolume`) are hard mounted by the Longhorn CSI plugin. Hard mounting allows Longhorn to persistently retry sending NFS requests, ensuring that I/Os do not fail even when the NFS server becomes unavailable for some time. I/Os resume seamlessly when the server regains connectivity or a replacement server is created.

This mechanism for guaranteeing data integrity, however, comes with some risk. To maintain stability, the Linux kernel might not allow unmounting of a file system until all pending I/Os are completed. This is a concern because the system cannot shut down until all file systems are unmounted. If the NFS server is unable to recover, the client nodes must undergo a forced reboot.

To mitigate the issue, you can perform the following actions:

- Existing volumes: Making changes to the `nfsOptions` field within `persistentvolume.spec.csi.volumeAttributes` is prohibited so you must upgrade Longhorn to `v1.4.4` or a later version. After upgrading, either `softerr` or `soft` is automatically applied to `nfsOptions` whenever RWX volumes are reattached.

- New volumes: Create a storage class with `nfsOptions`, and then create PVCs for RWX volumes using that specific storage class. By default, Longhorn uses the `softerr` or `soft` mount with a `timeo` value of "600" and a `retrans` value of "5". 

  Example:
  ```
  kind: StorageClass
  apiVersion: storage.k8s.io/v1
  metadata:
    name: longhorn-test
  provisioner: driver.longhorn.io
  allowVolumeExpansion: true
  reclaimPolicy: Delete
  volumeBindingMode: Immediate
  parameters:
    numberOfReplicas: "3"
    staleReplicaTimeout: "2880"
    fromBackup: ""
    fsType: "ext4"
    nfsOptions: "vers=4.1,noresvport,softerr,timeo=600,retrans=5"
  ``````

If the NFS server becomes unavailable, requests from NFS clients are sent according to the configured `retrans` value. During longer-duration events such as node power outages, the requests eventually fail and an NFS error (`ETIMEDOUT` or `EIO`) is returned to the calling application. Data loss may also occur.

For more information, see [issue #6655](https://github.com/longhorn/longhorn/issues/6655).